### ISSUE TITLE
The GitHub Action doesn't seem to work

### ISSUE THREAD

**Issue by directionless** [standardrb/standard-ruby-action#4]
# Bug Report 

## Describe the bug

This GitHub action does not seem to work. 

We have it installed in our rails app, and though there are files that would fail standardrb, the action passes them.

For example, a manual run:
```
$ standardrb lib/tasks/fill_webhook_event_subscriptions.rake
standard: Use Ruby Standard Style (https://github.com/standardrb/standard)
  lib/tasks/fill_webhook_event_subscriptions.rake:2:1: Layout/LeadingEmptyLines: Unnecessary blank line at the beginning of the source.
  lib/tasks/fill_webhook_event_subscriptions.rake:3:8: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
  lib/tasks/fill_webhook_event_subscriptions.rake:10:82: Style/RescueModifier: Avoid using `rescue` in its modifier form.
standard: Run `standardrb --fix` to fix up to 3 problems.
```

![Screenshot 2023-11-02 at 10 28 11](https://github.com/standardrb/standard-ruby-action/assets/179542/32f73753-df1b-41be-9daa-79038038ddeb)

# To Reproduce

# Expected behavior

standardrb should fail the job, and should create an annotation.

## Stacktrace

If applicable, add the action stacktrace. 

## Action Version & Workflow File
 
 - Version [e.g. 0.0.5]

### Workflow File

```yaml
  standardrb:
    needs: setup
    runs-on: ubuntu-latest
    permissions:
      contents: read
      checks: write
    if: github.event.pull_request
    steps:
      - uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5
      - name: StandardRB Linter
        uses: standardrb/standard-ruby-action/@0.0.5
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by directionless** [comment]
I'm having a hard time tracing the code here. The action starts a docker container, which installs gems, and then calls a service, which gather errors and then posts them to the GitHub API...

Is there a benefit over just running `standardrb --format github` ?

**Comment by robsdudeson** [comment]
Thanks for reporting this!  I've taken time to familiarize myself with the code.  I confirmed that the action always returns a success status regardless of violations.

My current thought would be to add an env var to toggle if we want to fail on _any_ violation.  From there it could be expanded to only fail at specific severity levels if there was a desire for that.  We'd check for the env var and violation count and pass/fail accordingly.

That being said, TIL about the [github formatter in Rubocop](https://docs.rubocop.org/rubocop/formatters.html#github-actions-formatter).  I'm going to explore this a bit more to see how it would work with the current implementation of the action.

**Comment by directionless** [comment]
It neither failed the PRs, not did it comment on the code. I can't tell if it did anything at all.

FWIW we stopped using this, and moved to a simple execution. So I may not be able to test much
```
      - uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5
      - name: setup ruby and gems
        uses: ruby/setup-ruby@5cfe23c062c0aac352e765b1b7cc12ea5255ccc4
        env:
          BUNDLE_GITHUB__COM: ${{ secrets.PAT_FOR_CLONES }}
        with:
          ruby-version: .ruby-version
          bundler-cache: true
          cache-version: 0
      - name: ls to check directory
        run: ls
      - name: standardrb
        run: bundle exec standardrb --format github
```

**Comment by camilopayan** [comment]
This should be fixed in 0.1.0
